java/http/vertx/deployment-tracing.yaml

```diff
@@ -26,21 +26,22 @@ spec:
             value: my-topic
           - name: SEND_INTERVAL
             value: "1000"
-          - name: JAEGER_SERVICE_NAME
-            value: java-http-vertx-producer
-          - name: JAEGER_AGENT_HOST
-            value: my-jaeger-agent
-          - name: JAEGER_SAMPLER_TYPE
-            value: const
-          - name: JAEGER_SAMPLER_PARAM
-            value: "1"
+          - name: MESSAGE_COUNT
+            value: "1000000"
+          - name: OTEL_SERVICE_NAME
+            value: http-vertx-otel
+          - name: OTEL_METRICS_EXPORTER
+            value: none
+          - name: OTEL_EXPORTER_OTLP_ENDPOINT
+            value: http://my-otlp-endpoint:4317
```

Comment:
Have you tried this Deployment after the latest change about the tracing system env var? I am not sure it works if you run it because you are missing to set ....

```yaml
- name: STRIMZI_TRACING_SYSTEM
  value: "opentelemetry"
```